iot/mqtt_example/cloudiot_mqtt_example_nodejs.js

```diff
@@ -255,8 +255,13 @@ let iatTime = parseInt(Date.now() / 1000);
 let client = mqtt.connect(connectionArgs);
 
 // Subscribe to the /devices/{device-id}/config topic to receive config updates.
+// Config updates are recommended to use QoS 1 (at least once delivery)
 client.subscribe(`/devices/${argv.deviceId}/config`, {qos: 1});
 
+// Subscribe to the /devices/{device-id}/commands/# topic to receive commands
+// commands is recommended to use QoS 0 (at most once delivery)
+client.subscribe(`/devices/${argv.deviceId}/commands/#`, {qos: 0});
```

Comment:
No action required but I'm using QoS 1 on other samples, it will retry until ack.